src/index.js

```diff
@@ -0,0 +1,151 @@
+import './style.css';
+
+const getDataFromLocalStorage = () => {
+  const todoList = localStorage.getItem('todoList');
+  return JSON.parse(todoList);
+};
+
+const updateView = () => {
```

Comment:
I would change the logic of how this function is applied by updating a single item instead of deleting everything and re-populating every time there is an update